jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -2375,6 +2375,15 @@ public boolean isInterface() {
  */
 public native boolean isPrimitive();
 
+/*[IF INLINE-TYPES]*/
```

Comment:
Is this fine or should I just leave it without the IF?

Reply:
that needs the ifdef